Makefile

```diff
@@ -146,7 +150,7 @@ fmt-check: ## Check formatting of Rego files
 	@opa fmt . --list --fail >/dev/null 2>&1
 
 .PHONY: ci
-ci: quiet-test opa-check conventions-check fmt-check ## Runs all checks and tests
+ci: coverage-report quiet-test opa-check conventions-check fmt-check ## Runs all checks and tests
```

Comment:
Should this be executed after all the other checks? (Not sure which order it will be executed now.)

Reply:
Though it makes it less useful for humans running make ci locally... :thinking: ...

Reply:
Not sure why would we care about coverage.json for a PR that is failing.

Reply:
Folk would get graphical representation of the coverage and see what cases need covering, we don’t get that currently